### DISCUSSION TITLE
Printing bug

### DISCUSSION THREAD

**Discussion by hyshen1** [sumatrapdfreader/sumatrapdf#3293] [General]
A long-term Sumatra user and lover here. Here's my finding: sometimes some pages are not correctly printed. I see blank stripes across the page. It's like one strip of printed contents followed by one blank strip with same height, alternately. I tested on latest version v3.4.6 and still see it. This is NOT seen on ALL pages, but some of them. Try with some text only pages first if you want to reproduce the issue.

On the same printer, I used Firefox to print the same file and it went very well, without the stripes.

**Comment by GitHubRulesOK** [comment]
Long time problem with some commercial grade drivers (often tied to an "optimize" setting) possibly related to a memory banding timer/timout but never been honed down as to what or why, never seen on simpler printers without staplers!

see issues that contain strap #919 #1762 OR band #1312 bands stripe stripes (sorry but search seems to be fixed words only)
